### DISCUSSION TITLE
Github actions trigger-workflow-and-wait stuck on 50% Received

### DISCUSSION THREAD

**Discussion by DianaWhist** [community/community#26729] [Actions]
<p>Hi,</p>
<p>I am  triggering a workflow with workflow_dispatch. Once from remote repo and once from inside the inside the same repo. Both times I get the same problem.</p>
<p>On first run it works properly, but on new dispatch, the step get stuck with</p>
<blockquote>
<p>Query params: event=workflow_dispatch&amp;status=in_progress</p>
</blockquote>
<p>This runs for long (Up to 30 minutes) and then fails with :</p>
<blockquote>
<p>jq: error (at :4): Cannot iterate over null (null)</p>
</blockquote>
<p>I do not use jq in the job or step, not in the one the triggers the remote workflow and not in the remote workflow.</p>
<p>The thing is, the workflow that is being triggered finishes it’s run with success in seconds. The remote workflow is deploys a cloud formation stack, the stack is already deployed and has no changes.</p>
<p>This is my step:</p>
<pre><code class="lang-auto">  Myjob:
    runs-on: ubuntu-latest
    needs:
      - SomeOtherJob    
    steps:
      - name: MyStepName
        continue-on-error: false
        uses: levon-kechichian-KR/trigger-workflow-and-wait-1@v1.6.0
        with:
          owner: OwnerName
          repo: RepoName
          github_token: XXXX
          workflow_file_name: MyDeploy.yaml
          client_payload: ''
          ref: master
          wait_interval: 30
          wait_workflow: true
</code></pre>
<p>Again, The remote workflow gets the trigger, runs and completes with success, by <code>MyStepName </code> Remains stuck.</p>
<p>This is the message I get every 30 sec:</p>
<blockquote>
<p>Sleeping for “30” seconds<br>
Using the following params to filter the workflow runs to get the triggered run id -<br>
Query params: event=workflow_dispatch&amp;status=in_progress<br>
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current<br>
Dload  Upload   Total   Spent    Left  Speed</p>
<p>0     0    0     0    0     0      0      0 --:–:-- --:–:-- --:–:--     0<br>
100    50  100    50    0     0    431      0 --:–:-- --:–:-- --:–:--   434</p>
</blockquote>
<p>Any ideas?</p>

**Comment by fage88** [comment]
<p>I am seeing similar issues with all the dispatch and wait actions in the market place.</p>
<p>I tried using 3 or 4 different ones today and they all seem to fail getting the run id back.</p>
<p>It looks like these were all working over the weekend as my workflows that are dependant on these actions all worked on Sunday but Monday and today they are not.</p>
<p>I’m hoping more people will notice and this will get escalated</p>

**Comment by DianaWhist** [comment, marked as answer]
<p>We downgraded today to levon-kechichian-KR/trigger-workflow-and-wait-1@v1.5.0 and it fixed the problem. Looks like a bug in 1.6.0.</p>
